sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -160,24 +204,15 @@ protected override async Task<bool> AcceptEvent (SessionId sessionId, string met
 		protected override async Task<bool> AcceptCommand (MessageId id, string method, JObject args, CancellationToken token)
 		{
 			switch (method) {
-			case "Target.attachToTarget": {
-					break;
-				}
-			case "Target.attachToBrowserTarget": {
-					break;
-				}
+
 			case "Debugger.getScriptSource": {
-					var script_id = args? ["scriptId"]?.Value<string> ();
-					if (script_id.StartsWith ("dotnet://", StringComparison.InvariantCultureIgnoreCase)) {
-						await OnGetScriptSource (id, script_id, token);
-						return true;
-					}
-					break;
+					var script = args? ["scriptId"]?.Value<string> ();
+					return await OnGetScriptSource (id, script, token);
```

Comment:
`script` here can be null, and that would end in a NRE in `ScriptSource.TryParse`. We can avoid calling `OnGetScriptSource` if `script == null`. 
Also, `ScriptSource.TryParse` should get `id == null`.

Reply:
fixed in a followup commit by fixing TryParse